python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -25,6 +25,101 @@ def register_launch_commands(self):
         Multiple commands may be registered based on the number of retrieved
         Software entities and their corresponding 'versions' field.
         """
+        # Retrieve the Software entities from SG and record how many were found.
+        sw_entities = self._sg_software_entities()
+        self._tk_app.log_info("Found (%d) Software entities." % len(sw_entities))
+        if not sw_entities:
+            # No commands to register if no entities were found.
+            return
+
+        # Resolve the app path and args field names for the current platform
+        app_path_field = "sg_%s_path" % self._platform_name
+        app_args_field = "sg_%s_args" % self._platform_name
+
+        # Collect a list of dictionaries that contain the information required
+        # to register a command with the current engine to launch a DCC.
+        register_cmds = []
+        for sw_entity in sw_entities:
+            self._tk_app.log_debug("Software Entity:\n%s" % pprint.pformat(sw_entity, indent=4))
+
+            # Parse the Software versions field to determine the specific list
+            # of versions to load. Assume the list of versions is stored as
+            # comma-separated in Shotgun.
+            app_versions = sw_entity["sg_versions"] or []
```

Comment:
i think if you do `app_versions = sw_entity["sg_versions"] or ""` you don't need the isinstance check ?